pkg/injector/service/injector.go

```diff
@@ -45,14 +45,13 @@ const (
 
 var log = logger.NewLogger("dapr.injector.service")
 
+// AllowedServiceAccountInfos default allowed service accounts
+// You can add other accounts which you want by adding them in helm chart file 'dapr_sidecar_injector/values.yaml'
+// helm chart value: allowedServiceAccounts or allowedServiceAccountsPrefixNames
 var AllowedServiceAccountInfos = []string{
 	"kube-system:replicaset-controller",
 	"kube-system:deployment-controller",
-	"kube-system:cronjob-controller",
```

Comment:
Those service accounts are default for many versions, why remove them?

Reply:
> Those service accounts are default for many versions, why remove them?

I have moved them to helm chart default values `allowedServiceAccounts` and not affect  pervious versions.

You can see the detail reason and design for this PR in this issue #6967